Rock/Web/UI/Controls/AttributeEditor.cs

```diff
@@ -1486,6 +1488,24 @@ protected void CreateFieldTypeDetailControls( int? fieldTypeId, bool recreate =
         }
 
         /// <summary>
+        /// Set the Subtitle of modal dialog
+        /// </summary>
+        /// <param name="attribute">The attribute.</param>
+        protected void SetSubTitleOnModal( Model.Attribute attribute )
+        {
+            Control parent = this.Parent;
+            while ( parent.GetType() != typeof( ModalDialog ) )
```

Comment:
@shivambareria Looks like this would cause an exception if the attribute editor was ever used without a parent modaldialog (parent would eventually be null).

There's an extension method that you can use instead for this: FirstParentControlOfType<T>.  That takes care of checking for null.